packages/medusa-plugin-segment/src/subscribers/order.js

```diff
@@ -163,7 +163,9 @@ class OrderSubscriber {
           ],
         })
 
-        const ret = await this.returnService_.retrieve(return_id)
+        const ret = await this.returnService_.retrieve(return_id, {
+          relations: ["items", "items.reason"],
+        })
```

Comment:
Technically I believe that these are eager loaded so not strictly necessary; however, if we were to change that at some point I think it is a very good idea that we are explicit about which relations the function depends on.